test/verify/check-kdump

```diff
@@ -24,9 +24,11 @@ from lib.constants import TEST_OS_DEFAULT
 class KdumpHelpers(testlib.MachineCase):
     def enableKdump(self):
```

Comment:
This helper contains so much unrelated junk.. `start_cockpit()` and the lines below are completely unnecessary (done through `self.login_and_go()` already), the `allow_restart()` should move into setUp() or the tests as that is unrelated. It's ok for now to always call the helper, although IMHO the tests would be easier to understand if they moved out the condition -- "if f38: enable_kdump()". but that would duplicate it between the two test classes, so keep it for now.

Reply:
Right, agreed. Let's do that in a future cleanup when F38 is dropped.

Reply:
I meant "keep the condition"; Please drop the other junk.